src/inc/profilepriv.h

```diff
@@ -146,12 +146,12 @@ GVAL_DECL(ProfControlBlock, g_profControlBlock);
 //
 //---------------------------------------------------------------
 
-#endif // defined(PROFILING_SUPPORTED_DATA) || defined(PROFILING_SUPPORTED)
-
 // This is the helper callback that the gc uses when walking the heap.
 BOOL HeapWalkHelper(Object* pBO, void* pv);
 void ScanRootsHelper(Object** ppObj, ScanContext *pSC, DWORD dwUnused);
 BOOL AllocByClassHelper(Object* pBO, void* pv);
+#endif // defined(PROFILING_SUPPORTED_DATA) || defined(PROFILING_SUPPORTED)
```

Comment:
Why does this ifdef need to be moved?  I would not have expected you to need to change this.